dex-it-common/src/main/scala/com/wavesplatform/dex/it/api/AsyncEnrichedApi.scala

```diff
@@ -3,33 +3,32 @@ package com.wavesplatform.dex.it.api
 import java.net.InetSocketAddress
 import java.util.UUID
 
-import com.softwaremill.sttp.{Request, SttpBackend}
+import sttp.client3._
 import com.typesafe.config.Config
 import play.api.libs.json.Reads
 
 import scala.concurrent.{ExecutionContext, Future}
 
 abstract class AsyncEnrichedApi[ErrorT: Reads](host: => InetSocketAddress)(implicit
   ec: ExecutionContext,
-  httpBackend: SttpBackend[Future, Nothing]
+  httpBackend: SttpBackend[Future, Any]
 ) {
 
   type R[EntityT] = Future[EnrichedResponse[ErrorT, EntityT]]
 
-  def mk[EntityT: Reads](req: Request[String, Nothing]): R[EntityT] =
-    httpBackend
-      .send[String](req.tag("requestId", UUID.randomUUID))
-      .map(EnrichedResponse(_, new EnrichedResponse.AsJson[ErrorT, EntityT]))
+  def mk[EntityT: Reads](req: Request[Either[String, String], Any]): R[EntityT] = req
+    .tag("requestId", UUID.randomUUID)
+    .contentType("application/json")
```

Comment:
This is not required, probably wrong in some cases. Content-Type is specified by BodySerializer

Reply:
fixed